docs/support-matrix.md

```diff
@@ -14,12 +14,13 @@ Ark supports a variety of storage providers for different backup and snapshot op
 
 Ark uses [Amazon's Go SDK][12] to connect to the S3 API. Some third-party storage providers also support the S3 API, and users have reported the following providers work with Ark:
 
-_Note that these providers are not regularly tested by the Ark team._
+_Note that these providers are not regularly tested by the Ark team. Some storage providers, like Quobyte, may need a different [signature algorithm version][15]._
```

Comment:
I think I'd move the note about Quobyte/v1 signature down to next to/under the Quobyte bullet.